### PR TITLE
courses completed

### PR DESCRIPTION
**Pull request by JulioContrerasH**

### CODE REVIEW

File: static/files/courses/Color_Vision.zip
Comment (gherardovarando) [inline static/files/courses/Color_Vision.zip:1]: maybe it is better if this files are in the content folder, together with the courses file/directory they belong to

File: static/files/courses/kernel_course.zip
Comment (gherardovarando) [inline static/files/courses/kernel_course.zip:1]: maybe it is better if this files are in the content folder, together with the courses file/directory they belong to

File: static/files/courses/ps_2014.pdf
Comment (gherardovarando) [inline static/files/courses/ps_2014.pdf:1]: maybe it is better if this files are in the content folder, together with the courses file/directory they belong to

File: static/files/courses/visual_percept.pptx
Comment (gherardovarando) [inline static/files/courses/visual_percept.pptx:1]: maybe it is better if this files are in the content folder, together with the courses file/directory they belong to

File: static/style/style.css
Comment (gherardovarando) [inline static/style/style.css:1]: ok

File: content/courses/color-vision-colorimetry.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Color Vision and Colorimetry"
+image: "/images/courses/color.webp"
+weight: 5
+draft: false
+---
+
+**Course Duration:** 30 hours
```
Comment (gherardovarando) [inline content/courses/color-vision-colorimetry.md:8]: maybe course duration and instructor could be part of the info in the frontmatter ?

File: content/courses/glass-training-material.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "The GLaSS Training Material Builds on the Global Lakes Use Cases"
+image: "/images/courses/CarwH7tWIAAn2ry.jpg-large.webp"
+weight: 10
+draft: false
+---
+
+**Course Duration:** N/A
```
Comment (gherardovarando) [inline content/courses/glass-training-material.md:8]: maybe course duration and instructor could be part of the info in the frontmatter ?

File: content/courses/color-vision-colorimetry.md
Comment (gherardovarando) [inline content/courses/color-vision-colorimetry.md:1]: maybe we should have subfolder for each course

File: themes/isp_uv/layouts/courses/list.html
Comment (gherardovarando) [inline themes/isp_uv/layouts/courses/list.html:1]: ok

**Review by gherardovarando — CHANGES_REQUESTED** [review]
I reviewed the changes and proposed some improvements.
Could you also update the README and provide a valid archetype for new course ?

File: static/files/courses/Color_Vision.zip
Comment (JulioContrerasH) [reply static/files/courses/Color_Vision.zip:1]: Yes, I agree, but there are very large files that weigh more than 100 MB, such as course zip files, where would it be convenient to store them for download?

Comment (JulioContrerasH) [reply static/files/courses/Color_Vision.zip:1]: There are several cases where it is not allowed to push

Comment (gherardovarando) [reply static/files/courses/Color_Vision.zip:1]: I think 100mb files should be still fine with GitHub no ?

Comment (JulioContrerasH) [reply static/files/courses/Color_Vision.zip:1]: There are errors in your weight upload, and I need to remove them in order to push to GitHub.

**Pull request merged**